webapp/graphite/settings.py

```diff
@@ -200,6 +204,14 @@
 if USE_LDAP_AUTH and LDAP_URI is None:
   LDAP_URI = "ldap://%s:%d/" % (LDAP_SERVER, LDAP_PORT)
 
+LOGIN_EXEMPT_URLS = (
+ r'^render',
+ r'^metrics/'
```

Comment:
I think this means that data can still be accessed, just the standard web interface is unavailable.  Is there a way to protect these endpoints too?